clients/admin-ui/src/pages/add-systems/new.tsx

```diff
@@ -0,0 +1,110 @@
+import { Box, Breadcrumb, BreadcrumbItem, Heading, Text } from "@fidesui/react";
+import type { NextPage } from "next";
+import NextLink from "next/link";
+import { useRouter } from "next/router";
+import React, { useMemo } from "react";
+
+import { useInterzoneNav } from "~/features/common/hooks/useInterzoneNav";
+import Layout from "~/features/common/Layout";
+import ConnectionTypeLogo from "~/features/datastore-connections/ConnectionTypeLogo";
+import DescribeSystem from "~/features/system/DescribeSystem";
+import SystemCatalog from "~/features/system/SystemCatalog";
+import { ConnectionSystemTypeMap } from "~/types/api";
+
+const CHOOSE_SYSTEM_COPY =
+  "Systems are anything that might store or process data in your organization, from a web application, to a database or data warehouse. Pick from common system types below or create a new type of system to get started.";
+const DESCRIBE_SYSTEM_COPY =
+  "Systems are anything that might store or process data in your organization, from a web application, to a database or data warehouse. Describe your system below to register it to the map. You may optionally complete data entry for the system using the additional tabs to navigate the sections.";
+
+type Step = "choose-system" | "describe-system";
+
+const Header = ({
+  step,
+  connector,
+}: {
+  step: Step;
+  connector?: ConnectionSystemTypeMap;
+}) => {
+  if (step === "choose-system") {
+    return (
+      <Heading fontSize="2xl" fontWeight="semibold" mb={2} data-testid="header">
+        Choose a type of system
+      </Heading>
+    );
+  }
+
+  return (
+    <Box display="flex" mb={2} alignItems="center" data-testid="header">
+      <ConnectionTypeLogo
+        data={connector ? connector.identifier : "ethyca"}
+        mr={2}
+      />
+      <Heading fontSize="2xl" fontWeight="semibold">
+        Describe your {connector ? connector.human_readable : "new"} system
+      </Heading>
+    </Box>
+  );
+};
+
+const NewSystem: NextPage = () => {
+  const { systemOrDatamapRoute } = useInterzoneNav();
+  const router = useRouter();
+  const { step, connectorType } = router.query;
```

Comment:
getting the connector type from the router query is how things are done on the connectors side. again, adhering to doing minimal work against this temporary API, I opted to just use the same query parameters as `ConnectionTypeList` was using so we could drop that component in and use it right away